internal/buildpackage/builder.go

```diff
@@ -42,22 +42,20 @@ func (p *PackageBuilder) Save(repoName string, publish bool) (imgutil.Image, err
 	}
 
 	stacks := p.buildpack.Descriptor().Stacks
-	if len(stacks) == 0 {
```

Comment:
Removed this since it's not possible to construct an invalid buildpack in our current code flow:

See https://github.com/buildpacks/pack/blob/master/internal/dist/buildpack.go#L205